Move contracts addresses info (#124)
threshold-network/solidity-contracts: docs/rfc-2-vending-machine.adoc

== Deployments and contract addresses

> Information about latest deployments and addresses of the contracts on Mainnet
> and Testnet can be found
> link:https://docs.threshold.network/extras/contract-addresses/goerli-testnet[here].

Reviewer: ```suggestion
Information about the latest deployments and addresses of the contracts can be found in the link:https://docs.threshold.network/extras/contract-addresses[Threshold Network documentation].
```

Author: 5c0fa0cef3b00a26e24639517790b11f8d4d1daf
